src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,263 @@
+/* eslint
```

Comment:
I don't think we have 100% consensus yet, but I want to see us try something anyway.  The least controversial one to do imho is the JS formatting.  Let's just do it.

Also, please remove this eslint comment, we can't do this in this file for reasons we've discussed.